apps/meteor/ee/server/patches/verifyContactChannel.ts

```diff
@@ -29,20 +29,10 @@ async function _verifyContactChannel(
 		session.startTransaction();
 		logger.debug({ msg: 'Start verifying contact channel', contactId, visitorId, roomId });
 
-		await LivechatContacts.updateContactChannel(
-			{
-				visitorId,
-				source: room.source,
-			},
-			{
-				verified: true,
-				verifiedAt: new Date(),
-				field,
-				value: value.toLowerCase(),
-			},
-			{},
-			{ session },
-		);
+		const updater = LivechatContacts.getUpdater();
+		LivechatContacts.setVerifiedUpdateQuery(true, updater);
+		LivechatContacts.setFieldAndValueUpdateQuery(field, value, updater);
```

Comment:
Aren't we forgetting the lower case here?

Reply:
hmmm not sure I got you... which lower case?